cookbook/serializer.rst

```diff
@@ -101,4 +124,96 @@ Here is an example on how to load the
         $definition->addTag('serializer.normalizer');
         $container->setDefinition('get_set_method_normalizer', $definition);
 
-.. _JMSSerializerBundle: http://jmsyst.com/bundles/JMSSerializerBundle
+Using Serialization Groups Annotations
+--------------------------------------
+
+.. versionadded:: 2.7
+    The serialization group system has been added in Symfony 2.7.
+
+Enable :ref:`serialization groups annotation<component-serializer-attributes-groups>`
+with the following configuration:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/config.yml
+        framework:
+            # ...
+            serializer:
+                enable_annotations: true
+
+    .. code-block:: xml
+
+        <!-- app/config/config.xml -->
+        <framework:config>
+            <!-- ... -->
+            <framework:serializer enable-annotations="true" />
+        </framework:config>
+
+    .. code-block:: php
+
+        // app/config/config.php
+        $container->loadFromExtension('framework', array(
+            // ...
+            'serializer' => array(
+                'enable_annotations' => true,
+            ),
+        ));
+
+Enabling the Metadata Cache
+---------------------------
+
+.. versionadded:: 2.7
+    Serializer metadata has been added in Symfony 2.7.
+
+Metadata used by the Serializer component such as groups can be cached to
+enhance application performance. Any service implementing ``Doctrine\Common\Cache\Cache``
+interface can be used.
+
+A service leveraging `APCu`_ (and APC for PHP < 5.5) is builtin.
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/config_prod.yml
+        framework:
+            # ...
+            serializer:
+                cache: serializer.mapping.cache.apc
+
+    .. code-block:: xml
+
+        <!-- app/config/config_prod.xml -->
+        <framework:config>
+            <!-- ... -->
+            <framework:serializer cache="serializer.mapping.cache.apc" />
+        </framework:config>
+
+    .. code-block:: php
+
+        // app/config/config_prod.php
+        $container->loadFromExtension('framework', array(
+            // ...
+            'serializer' => array(
+                'cache' => 'serializer.mapping.cache.apc',
+            ),
+        ));
+
+Going Further with the Serializer Component
+-------------------------------------------
+
+`DunglasApiBundle`_ provides an API system supporting `JSON-LD`_ and `Hydra Core Vocabulary`_
+hypermedia formats. It is built on top of the Symfony framework and its Serializer
+component.
+It provides custom normalizers and a custom encoder, custom metadata and a caching
```

Comment:
There is no need for the newline here. You would have to add a blank line if you wanted to start a new paragraph.